src/magic_folder/test/test_snapshot.py

```diff
@@ -99,10 +106,22 @@ def setUp(self):
         self.alice = create_local_author("alice")
         self.stash_dir = mktemp()
         os.mkdir(self.stash_dir)
+
+        # create a magicfolder db
+        self.tempdb=abspath_expanduser_unicode(unicode(mktemp()))
+        os.mkdir(self.tempdb)
+        dbfile= abspath_expanduser_unicode(u"test_snapshot.sqlite", base=self.tempdb)
+        self.db = magicfolderdb.get_magicfolderdb(dbfile, create_version=(magicfolderdb.SCHEMA_v1, 1))
+
+        self.failUnless(self.db, "unable to create magicfolderdb from %r" % (dbfile,))
+        self.failUnlessEqual(self.db.VERSION, 1)
```

Comment:
I think in new code we should use `.format()`. That said, I'm not sure if we've gotten consensus around any of this kind of string handling stuff :/